raiden/tests/integration/test_matrix_transport.py

```diff
@@ -622,3 +633,96 @@ def test_monitoring_global_messages(
     assert ms_room.send_text.call_count == 1
     transport.stop()
     transport.get()
+
+
+def test_pfs_global_messages(
+        local_matrix_servers,
+        private_rooms,
+        retry_interval,
+        retries_before_backoff,
+        monkeypatch,
+):
+    """
+    Test that `update_pfs` from `RaidenEventHandler` sends balance proof updates to the global
+    PATH_FINDING_BROADCASTING_ROOM room on Send($BalanceProof)* events, i.e. events, that send
+    a new balance proof to the channel partner.
+    """
+    transport = MatrixTransport({
```

Comment:
There's a `matrix_transports` fixture that gives you default transport(s) for the number of requested matrix servers (or optionally more).

(You'd need to add the `PATH_FINDING_BROADCASTING_ROOM` in there though.)